modules/lunamediaBidAdapter.md

```diff
@@ -29,7 +29,8 @@ var displayAdUnit = [
         bidder: 'lunamedia',
         params: {
             pubid: '121ab139faf7ac67428a23f1d0a9a71b',
-            placement: 1234
+			placement: 1234,
+			size="320x50"
```

Comment:
Change `=` to `:`

Same change on line 57

Reply:
changes are done.